### PR TITLE
Suppress warnings

### PR DESCRIPTION
**Pull request by dmdashenkov**
In this PR we add Java renderers that generate annotations on the top-level declarations.

- `SuppressRenderer` adds `@SuppressWarnings`. The warnings can be specified via a configuration. By default, all warnings are suppressed.
- `GenerateGenerated` adds `@Generated`.

The `SuppressConfig` type, which configures the `SuppressRenderer` also teaches the user how to create and combine different configurations for different renderers.

### CODE REVIEW

**Comment by dmdashenkov** [comment]
@armiol, PTAL.

File: codegen/java/src/main/kotlin/io/spine/protodata/codegen/java/generado/GenerateGenerated.kt [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+ * in the source set.
+ *
+ * @see io.spine.protodata.codegen.java.suppress.SuppressRenderer
+ */
+public class GenerateGenerated : JavaRenderer() {
+
+    override fun render(sources: SourceSet) {
+        sources.forEach {
+            it.at(BeforePrimaryDeclaration).add(
+                "@${Generated::class.qualifiedName}(\"by the Protobuf Compiler and/or ProtoData\")"
```
Comment (armiol) [inline codegen/java/src/main/kotlin/io/spine/protodata/codegen/java/generado/GenerateGenerated.kt:54]: Please reconsider this "and/or". It's really weird.

File: codegen/java/src/test/kotlin/io/spine/protodata/codegen/java/generado/GenerateGeneratedTest.kt [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            renderers = listOf(PrintBeforePrimaryDeclaration(), GenerateGenerated()),
+            sourceSet = sourceSet,
+            request = CodeGeneratorRequest.getDefaultInstance()
+        )()
+        val code = sourceSet
+            .file(Path(JAVA_FILE))
+            .code()
+        assertThat(code)
+            .contains("""
+                @javax.annotation.Generated("by the Protobuf Compiler and/or ProtoData")
```
Comment (armiol) [inline codegen/java/src/test/kotlin/io/spine/protodata/codegen/java/generado/GenerateGeneratedTest.kt:53]: And is it possible to re-use the string literal from its origin?

**Review by armiol — APPROVED** [review]
@dmdashenkov LGTM with a couple of comments to address.

**Pull request merged**